src/lib/Behat/Component/UniversalDiscoveryWidget.php

```diff
@@ -194,6 +188,28 @@ public function selectInSearchResults(string $name): void
             ->click();
     }
 
+    public function clickCreateNewButton(): void
+    {
+        $this->getHTMLPage()->find($this->getLocator('createNewButton'))->click();
+    }
+
+    public function verifyCreateOnTheFlyFormIsLoaded(): void
+    {
+        $this->switchIntoUDWIframe('createIframe');
+        $this->getHTMLPage()->setTimeout(self::LONG_TIMEOUT)->find($this->getLocator('createOnTheFlyForm'))->assert()->isVisible();
+    }
+
+    public function switchIntoUDWIframe(string $iframeSelectorIdentifier): void
```

Comment:
```suggestion
    private function switchIntoUDWIframe(string $iframeSelectorIdentifier): void
```
This method should not be called outside the UDW Component, so i've make it private.